components/reference-data/src/main/java/org/datacleaner/beans/filter/DictionaryFilter.java

```diff
@@ -47,20 +52,39 @@
     @Configured
     Dictionary dictionary;
 
+    @Provided
+    DataCleanerConfiguration configuration;
+
+    private DictionaryConnection dictionaryConnection;
+
     public DictionaryFilter() {
     }
 
-    public DictionaryFilter(InputColumn<String> column, Dictionary dictionary) {
+    public DictionaryFilter(InputColumn<String> column, Dictionary dictionary, DataCleanerConfiguration configuration) {
```

Comment:
Any reason to have this? It's only used by it's test, which is in the same package, so the properties should be available through the default scope anyway.